gem_city.rb

```diff
@@ -0,0 +1,43 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+  attr_accessor :thieves, :officers
+
+  def initialize
+    @thieves = 5
+    @officers = 1
+    @population = 50
+  end
+
+  def happiness_of_town
+    # happiness is random... people don't know what they want!
+    happiness_vals = (1..@population).collect do
+      rand((100 - successful_crime_rate)..100)
+    end
+    happiness = happiness_vals.reduce(0, :+)
+    happiness / 100
+  end
+
+  def no_crimes_can_occur?
+    @thieves <= 0 || @officers > @thieves
```

Comment:
It was complaining about me not understanding the intent until just now.  I _think_ that I have it now.